NitroxModel/Networking/INitroxConnectioHelper.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace NitroxModel.Networking
+{
+    public static class IConnectionInfoHelper
```

Comment:
Class and file name is different. Also shouldn't contain "I" as it's no interface.